plugins/modules/net_tools/dnsimple.py

```diff
@@ -184,12 +352,10 @@ def main():
     )
 
     if not HAS_DNSIMPLE:
-        module.fail_json(msg=missing_required_lib('dnsimple'), exception=DNSIMPLE_IMP_ERR)
-
-    if LooseVersion(dnsimple_version) < LooseVersion('1.0.0'):
-        module.fail_json(msg="Current version of dnsimple Python module [%s] uses 'v1' API which is deprecated."
-                             " Please upgrade to version 1.0.0 and above to use dnsimple 'v2' API." % dnsimple_version)
+        module.fail_json(msg=missing_required_lib('dnsimple'), exception=str(DNSIMPLE_IMP_ERR))
```

Comment:
`DNSIMPLE_IMP_ERR` is now a list; `exception` should be a string with **one** exception traceback.

Reply:
right, I was not sure how to handle this since now we try to import the dnsimple library in two blocks and theoretically both could fail so I thought it would be nice if we had both exception tracebacks in the fail message. But I guess in that case I will just set `DNSIMPLE_IMP_ERR` to a string again which then only contains the exception traceback of the last failed import block.

Reply:
Both using the first or last would be fine. (Using the first - trying to import 2.0.0+ - might nudge people more to using the latest version :) )

Reply:
Good point :)
Let's use the first then